main.tf

```diff
@@ -0,0 +1,175 @@
+terraform {
+  required_version = "~> 1.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+    template = {
```

Comment:
I think the template provider is deprecated- pls avoid usage if possible. Also doesn't build on M1.